src/api.ts

```diff
@@ -120,6 +120,50 @@ export interface QuoteRequest extends ToolConfiguration {
   referrer?: string
 }
 
+export interface QuoteContractCallRequest extends ToolConfiguration {
```

Comment:
I think as this is a quote for a contract call tx, `ContractCallQuoteRequest` is a bit more fitting. But not a very strong opinion